pages/data-design/branches/pulling-pushing-items-between-branches.mdx

```diff
@@ -0,0 +1,113 @@
+
+## Pulling items from another branch (coming in March 2025)
+
+Soon it will be possible to search and get results from other branches. Those item can be pulled to the branch the user is on, and can be used on that branch.
+
+## Merging branches where items have been pushed or pulled
+
+When either the source or destination branch where items have been pushed or pulled are merged to main, you will be prompted to pull main on the branch where the items still remain.
+As the ID of the items are the same, the items exist on main and there is no need to keep them specifically on the other branch. 
+
+
+## Property changes related to event
+
+When an event or property bundle is selected to push to another branch, the review modal will show a list of properties related to the event or property bundle.
+By default, all related properties and their changes are kept on the event or property bundle.
+
+For events, it's possible to toggle whether the related properties and their changes are pushed too. The related property changes are the following:
+- Any **new properties** that were added to the selected item and **do not exist** on the destination branch
+    - When toggled off on events, the property will be excluded on the destination branch
+- Any **existing properties** that were added to the selected item and are **not on the event** on the destination branch
+    - When toggled off on events, any changes to the property will be excluded on the destination branch
+- Any **existing properties** on the selected item have been modified such that they are **different** on the destination branch
+    - When toggled off on events, any changes to the property will be excluded on the destination branch
+
+<center><img
+  src="/docs/images/pulling-pushing-items/toggle-related-property-changes.png"
+  width="60%"
+  alt="Toggle related property changes"
+/></center>
+
+## What can be pushed and pulled and from where?
+
+Pushing (and soon pulling) is currently available for the following items:
+- Events
+- Properties
+- Property bundles
+
+Pushing is available for new and updated items from the following views:
+- Tracking plan changes (button on each item that can be pushed)
+
+## Pushing and pulling use cases
+
+Pushing and pulling has several use cases. Some examples are below.
+
+### Breaking down a large branch
+
+It often happens when designing data, that a branch becomes quite big, containing a lot of event and property changes.
+Then it's helpful to be able to break that branch into more manageable chunks, both to make review and implementation easier.
+
+That is now possible by pushing items from the diff view to different branches. 
+
+### One team is ready to implement but not others
+
+Commonly, Avo branches need to be implemented by more than one team. Most of the time teams are not able to implement in parallel, so it can be helpful to break the branches down for each team.
```

Comment:
Maybe frame this slightly differently IDK
```suggestion
Commonly, Avo branches need to be implemented by more than one team. In many cases teams are not able to implement in parallel, so it can be helpful to break the branches down for each team.
```